src/lib/autocomplete/autocomplete-trigger.ts

```diff
@@ -47,18 +45,6 @@ import {coerceBooleanProperty} from '@angular/cdk/coercion';
 import {MatAutocompleteOrigin} from './autocomplete-origin';
 
 
-/**
- * The following style constants are necessary to save here in order
- * to properly calculate the scrollTop of the panel. Because we are not
- * actually focusing the active item, scroll must be handled manually.
- */
-
-/** The height of each autocomplete option. */
-export const AUTOCOMPLETE_OPTION_HEIGHT = 48;
-
-/** The total height of the autocomplete panel. */
-export const AUTOCOMPLETE_PANEL_HEIGHT = 256;
```

Comment:
Since these ended up being exported from the package, deleting them will be a breaking change.